.gitignore

```diff
@@ -24,3 +24,6 @@ __pycache__/
 # CMake
 CMakeLists.txt
 *.cmake
+
+gdal/packages*
```

Comment:
those additions seem to be particular to your setup ?

Reply:
Not that particular, that's where NuGet package manager places it's files.

Reply:
ok, can you add a comment in .gitignore then ?